src/pages/Shikshalokam Mitra/MainPage.jsx

```diff
@@ -38,11 +40,14 @@ function MainPage() {
     const audioRef = useRef();
 
     useEffect(()=>{
-        setUserDetail((values)=>({
-            ...values,
-            name: localStorage.getItem("name") || "Rahul",
-            image: localStorage.getItem("image") || "",
-        }))
+        localStorage.setItem("name", "Rahul")
+        localStorage.setItem("image", "")
+        localStorage.setItem("email", "rahul@example.com")
+        setUserDetail({
```

Comment:
If we are setting the value again here, why are we taking the values from local in line 26, 27, 28